Resources/Prototypes/_LostParadise/Loadouts/Jobs/security.yml

```
@@ -1,37 +1,767 @@
+    - !type:CharacterDepartmentTimeRequirement
+      department: Security
+      min: 600000 # 1 hours
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutWeaponSecurity
+    - !type:CharacterDepartmentRequirement
+      departments:
+        - Security
+  items:
+    - LPPWeaponRevolverAmina
+
+#Пульки
```

yamllint

[warning] missing starting space in comment

(comments)